plugins/ouctl.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ouctl
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/tremolosecurity/openunison-control
+  shortDescription: "Deploy OpenUnison for authentication and automation"
+  description: Deploys OpenUnison, see https://openunison.github.io/deployauth/ for details
```

Comment:
The new description reads like the plugin is meant to be invoked one-time. Is this the case? Or are there ongoing management subcommands?

Reply:
in a single cluster, you can use it to both deploy and update openunison (it does error checking and timing of deployment between multiple helm charts).  In a multi cluster environment, it's used for onboarding new clusters so it would be called for each new cluster to enable authentication on it.